### PR TITLE
Update README for all samples

### LINKED ISSUE #11: add README.md for all samples
Each README.md should feature:
- a brief description of the sample
- a screenshot
- dependencies
- generic studio + c++ getting started instructions
- sample specific instructions when applicable
- support links
- license

https://github.com/googlesamples/android-ndk/tree/master/hello-jni is a good example.

### PR DESCRIPTION
**Pull request by proppy**
Fixes #11

/cc @ggfan @hak 

@thedmail do you have time to take a look at this and proof read the prose (esp. for the first paragraph of each sample)? Not being an english native speaker, I often have doubt about my formulation.

### CODE REVIEW

File: Teapot/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Teapot
+======
+Teapot is an Android C++ sample that draw a Teapot mesh using GLES 2.0 API and [NativeActivity](http://developer.android.com/reference/android/app/NativeActivity.html).
```
Comment (marcacohen) [inline Teapot/README.md]: s/draw/draws/

File: bitmap-plasma/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Bitmap Plasma
+=============
+Bitmap Plasma is an Android sample that uses JNI to renders a plasma effect in a Android [Bitmap](http://developer.android.com/reference/android/graphics/Bitmap.html) from C code.
```
Comment (marcacohen) [inline bitmap-plasma/README.md]: s/renders/render/

File: hello-gl2/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+Hello GL2
+=========
+Hello GL2 is an Android C++ sample that draw a triangle using GLES 2.0 API.
```
Comment (marcacohen) [inline hello-gl2/README.md]: s/draw/draws/

File: native-activity/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+Native Activity
+===============
+Native Activity is an Android sample that initializes a GLES 2.0 context and read accelerometer value from C code using [Native Activity](http://developer.android.com/reference/android/app/NativeActivity.html).
```
Comment (marcacohen) [inline native-activity/README.md]: s/read/reads/

File: native-audio/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+Native Audio
+============
+Native Audio is an Android sample that play and record sounds with the C++ OpenSLES API using JNI.
```
Comment (marcacohen) [inline native-audio/README.md]: plays and records

File: native-media/README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+Native Media
+============
+Native Media is an Android sample that uses OpenMAX AL to a play video.
```
Comment (marcacohen) [inline native-media/README.md]: "to play a video"?

File: native-plasma/README.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+Native Plasma
+=============
+Native Plasma is an Android sample that render a plasma effect in a Bitmap from C code using [Native Activity](http://developer.android.com/reference/android/app/NativeActivity.html).
```
Comment (marcacohen) [inline native-plasma/README.md]: renders

File: san-angeles/README.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+San Angeles
+===========
+San Angeles is a Android port of a demo that uses GLES C/API to render a procedural scene.
+
+See the original [README](app/src/main/jni/README.txt) more details about the original GLES port.
```
Comment (marcacohen) [inline san-angeles/README.md]: *for* more details about...

File: sensor-circle/README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
+Sensor-Graph
+============
+Sensor graph is a C++ Android sample that read current accelerometer values and draw them using OpenGL.
```
Comment (marcacohen) [inline sensor-circle/README.md]: reads/draws

File: endless-tunnel/README.md [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+For more information about this library and its license, please see
+Tunnel/jni/glm/readme.txt and Tunnel/jni/glm/copying.txt.
+
+Acknowledgements
+----------------
+Most of this code was written by [Bruno Oliveira](https://plus.google.com/+BrunoOliveira).
+
+Walkthrough
+-----------
+Source code are under app, it only contains C++ code, which is
```
Comment (marcacohen) [inline endless-tunnel/README.md]: "is under app. It only containers..."

File: native-activity/README.md (hunk 4 above)
Comment (ryanseys) [inline native-activity/README.md]: Also s/value/data

File: san-angeles/README.md (hunk 8 above)
Comment (ryanseys) [inline san-angeles/README.md]: *an* Android

File: endless-tunnel/README.md (hunk 10 above)
Comment (ryanseys) [inline endless-tunnel/README.md]: s/are/is needs to be corrected but I think @marcacohen typo'd at the end: "contains" is correct, not containers.

Comment (marcacohen) [inline endless-tunnel/README.md]: yes, typo, thx Ryan

File: endless-tunnel/README.md [hunk 11]
```diff
@@ -0,0 +1,205 @@
+Endless Tunnel
+==============
+Endless Tunnel is a sample game that shows how to:
+- use the Android Studio C++ support
+- implement a game using Android native glue
+- implement joystick support, including robust DPAD navigation for no-touch screens
```
Comment (ryanseys) [inline endless-tunnel/README.md]: non-touch screens

File: endless-tunnel/README.md [hunk 12]
```diff
@@ -0,0 +1,205 @@
...
+and has to be torn down in KillGraphics().
+
+The engine_init_display function is where we set up OpenGL
+for our game, and call StartGraphics() on the active scene.
+The engine_term_display is where we call KillGraphics() on the active
+scene.
+
+Input arrives by way of the engine_handle_input function, which
+does some basic input classification and delivers the input to
+the scene manager. Incidentally, here we also synthethise DPAD events based
```
Comment (ryanseys) [inline endless-tunnel/README.md]: typo: synthesise

File: Teapot/README.md (hunk 1 above)
Comment (proppy) [inline Teapot/README.md]: Done.

File: bitmap-plasma/README.md (hunk 2 above)
Comment (proppy) [inline bitmap-plasma/README.md]: Done.

File: hello-gl2/README.md (hunk 3 above)
Comment (proppy) [inline hello-gl2/README.md]: Done.

File: native-activity/README.md (hunk 4 above)
Comment (proppy) [inline native-activity/README.md]: Done.

File: native-audio/README.md (hunk 5 above)
Comment (proppy) [inline native-audio/README.md]: Done.

File: native-media/README.md (hunk 6 above)
Comment (proppy) [inline native-media/README.md]: Done.

File: native-plasma/README.md (hunk 7 above)
Comment (proppy) [inline native-plasma/README.md]: Done.

File: san-angeles/README.md (hunk 8 above)
Comment (proppy) [inline san-angeles/README.md]: Done.

File: sensor-circle/README.md (hunk 9 above)
Comment (proppy) [inline sensor-circle/README.md]: Delete.

File: endless-tunnel/README.md (hunk 10 above)
Comment (proppy) [inline endless-tunnel/README.md]: Done.

File: endless-tunnel/README.md (hunk 11 above)
Comment (proppy) [inline endless-tunnel/README.md]: Done.

File: endless-tunnel/README.md (hunk 12 above)
Comment (proppy) [inline endless-tunnel/README.md]: Done.

**Comment by proppy** [comment]
PTAL

**Comment by ryanseys** [comment]
LGTM

**Pull request merged**